Update README.md (#89)
JuliaSmoothOptimizers/MUMPS.jl: README.md

* [ ] Support distributed matrices / vectors
* [ ] User-selected permutation
* [X] Out-of-core option (in [73e829b](https://github.com/JuliaSmoothOptimizers/MUMPS.jl/commit/73e829b52fe3d20c70c2733607ba9820cda03ed6#diff-d41d8cd98f00b204e9800998ecf8427e))
* [X] Determinant (in [73e829b](https://github.com/JuliaSmoothOptimizers/MUMPS.jl/commit/73e829b52fe3d20c70c2733607ba9820cda03ed6#diff-d41d8cd98f00b204e9800998ecf8427e))
* [X] Compute entries of the inverse (in [002b34eb](https://github.com/JuliaSmoothOptimizers/MUMPS.jl/commit/002b34eb861657118c05fa08f2b0eb5aababe22c))
* [X] Control iterative refinement (in [73e829b](https://github.com/JuliaSmoothOptimizers/MUMPS.jl/commit/73e829b52fe3d20c70c2733607ba9820cda03ed6#diff-d41d8cd98f00b204e9800998ecf8427e))
* [X] Obtain a Schur complement (in [002b34eb](https://github.com/JuliaSmoothOptimizers/MUMPS.jl/commit/002b34eb861657118c05fa08f2b0eb5aababe22c))
* [X] Solve with sparse right-hand sides (in [002b34eb](https://github.com/JuliaSmoothOptimizers/MUMPS.jl/commit/002b34eb861657118c05fa08f2b0eb5aababe22c))

> * [ ] Sequential, version with no MPI requirement

Reviewer: I opened issues to replace this list.

> To use your custom MUMPS, set the environmental variables `JULIA_MUMPS_LIBRARY_PATH`

Reviewer: ```suggestion
To use your custom MUMPS, set the environment variable `JULIA_MUMPS_LIBRARY_PATH`
```

> to point the shared library. Note that **MUMPS** version 5.5.1 is needed.

Reviewer: ```suggestion
to point to the shared library before `using MUMPS`. Note that **MUMPS** version 5.5.1 is needed.
```

Reviewer: Sometimes MUMPS is bold and sometimes not.

> **Very important note: you must set these environment variables before
> calling `using MUMPS` in every Julia session.**
> 

Reviewer: ```suggestion
```

> 
> Alternatively, you can set these permanently through your operating system.

Reviewer: ```suggestion
```
